cpu/cc2538/include/periph_cpu.h

```diff
@@ -167,6 +168,44 @@ typedef struct {
 } uart_conf_t;
 /** @} */
 
+/**
```

Comment:
Without `#ifndef DOXYGEN` the UART API documentation is broken i.e. you have redundant defines. 

What about excluding the redefines for now till there is a final solution for this issue.

Reply:
Done